support/modules-artifact-gen/directory-artifact-gen

```diff
@@ -9,14 +9,18 @@ Simple tool to generate Mender Artifact suitable for directory Update Module
 
 Usage: $0 [options] directory [-- [options-for-mender-artifact] ]
 
-    Options: [ -n|artifact-name -t|--device-type -d|--dest-dir -o|--output_path -h|--help ]
-
-        --artifact-name     - Artifact name
-        --device-type       - Target device type identification (can be given more than once)
-        --dest-dir          - Target destination directory where to deploy the update
-        --output-path       - Path to output file. Default: directory-artifact.mender
-        --help              - Show help and exit
-        directory           - File tree to bundle in the update
+    Options: [ -n|artifact-name -t|--device-type -d|--dest-dir --software-name --software-value --software-filesystem -o|--output_path -h|--help ]
+
+        --artifact-name       - Artifact name
+        --device-type         - Target device type identification (can be given more than once)
+        --dest-dir            - Target destination directory where to deploy the update
+        --software-name       - Name of the key to store the software version: rootfs-image.directory.version,
+                                instead of rootfs-image.single-file.version
```

Comment:
Copy paste error? I guess this should be `NAME` and `directory`, like the other two.

Reply:
Ops, sorry, fixed.